setup.cfg

```diff
@@ -3,7 +3,6 @@ universal = 1
 
 [metadata]
 name = snowflake-sqlalchemy
-version = 1.4.6
```

Comment:
Should this be updated to 1.4.7 instead?

Reply:
the version number is now in `version.py`